styles/Home.module.css

```diff
@@ -3,276 +3,56 @@
   flex-direction: column;
```

Comment:
Remove entire file and replace with tailwind css classe

Reply:
Done as stated